starlite/router.py

```diff
@@ -96,7 +103,7 @@ def route_handler_method_map(self) -> Dict[str, Union[WebsocketRouteHandler, Dic
     @staticmethod
     def map_route_handlers(
         value: Union[Controller, BaseRouteHandler, "Router"],
-    ) -> ItemsView[str, Union[WebsocketRouteHandler, ASGIRoute, Dict[HttpMethod, HTTPRouteHandler]]]:
+    ) -> ItemsView[str, Union[WebsocketRouteHandler, WSMessageHandler, ASGIRoute, Dict[HttpMethod, HTTPRouteHandler]]]:
```

Comment:
i think its time to introduce some type aliases for this 😉

Reply:
hah yeh they are big

Reply:
I've done this one, I don't think I have the types right for route_handler_method_map() either, but I'll add a review for that.